src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -37,4 +37,30 @@
     <Reference Include="mscorlib" />
     <Reference Include="System" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <_packageTargetsFile>$(IntermediateOutputPath)$(AssemblyName).targets</_packageTargetsFile>
+  </PropertyGroup>
+
+  <Target Name="GeneratePackageTargetsFile" 
+          Condition="'$(TargetFramework)' == 'net461'"
+          Inputs="$(MSBuildAllProjects)"
+          Outputs="$(_packageTargetsFile)"
+          BeforeTargets="GetFilesToPackage">
+    <PropertyGroup>
+      <_packageTargetsFileContent><![CDATA[<Project>
+  <!-- ResolveAssemblyReferences will never see the assembly reference embedded in the resources type,
+       force a binding redirect ourselves so that we'll always unify to the System.Resources.Extensions
+       version provided by this package -->
+  <ItemGroup>
+    <SuggestedBindingRedirects Include="$(AssemblyName), Culture=neutral, PublicKeyToken=$(PublicKeyToken)" MaxVersion="$(AssemblyVersion)" />
```

Comment:
I decided to generate this file since it depends on AssemblyVersion.  That way we don't need to "remember" to update another place when AssemblyVersion changes.